crates/miden-lib/asm/shared/util/account_id.masm

```diff
@@ -87,7 +87,7 @@ end
 #! - other_acct_id_{prefix,suffix} are the prefix and suffix felts of the other account ID to
 #!   compare against.
 #! - is_id_equal is a boolean indicating whether the account IDs are equal.
-export.is_equal
+export.is_id_equal
```

Comment:
Since this is called as `account_id::is_equal`, is the "id" actually necessary? (Same for `validate_id`).

Reply:
Ah, right, at some point of refactoring I thought that these names were lacking the `id` "identity", but I agree, we already have it in the module name. I'll roll back this renaming.